wbooks/babel.config.js

```diff
@@ -1,3 +1,22 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
-};
+  plugins: [
+    [
+      'module-resolver',
+      {
+        root: ['./src'],
+        alias: {
+          '@app': './src/app',
+          '@assets': './src/assets',
+          '@config': './src/config',
+          '@constants': './src/constants',
+          '@propTypes': './src/propTypes',
+          '@redux': './src/redux',
+          '@scss': './src/scss',
+          '@services': './src/services',
+          '@utils': './src/utils'
```

Comment:
Remove proptypes and scss, you won't use them
And add them to `.tsconfig.json` too, in `paths` like this:

```
...
"paths": {
      "@app/*": ["src/app/*"],
      "@components/*": ["src/app/components/*"],
      ...
},
...
```

Reply:
I should also add interfaces right?

Reply:
Also I didn't create the assets folder, I guess we should add that to the folder structure card